### PR TITLE
Support buttwoo-v1

### PR DESCRIPTION
**Pull request by arj03**
Title says it all. Support [ssb-butt2](https://github.com/arj03/ssb-butt2-format). I'm not 100% certain on the name so I'll leave this as draft.

### CODE REVIEW

**Comment by staltz** [comment]
@arj03 Before making this official, don't forget to send a PR to `ssbc/ssb-uri-spec`

**Comment by arj03** [comment]
Good point. 

Btw. I just got a great idea that I'm trying out for this new format.

**Comment by arj03** [comment]
@staltz and properly decide on the name. I updated the spec to use the buttwoo name: https://github.com/arj03/ssb-buttwoo-spec. I'll also move the spec and code into ssbc. It's really great you have moved things in there, much easier to find stuff :)

File: src/index.ts [hunk 1]
```diff
@@ -3,11 +3,13 @@ import {BlobId, FeedId, MsgId} from 'ssb-typescript';
 type FeedTF =
   | ['feed', 'ed25519']
   | ['feed', 'bendybutt-v1']
-  | ['feed', 'gabbygrove-v1'];
+  | ['feed', 'gabbygrove-v1']
+  | ['feed', 'butt2-v1'];
```
Comment (staltz) [inline src/index.ts:7]: `buttwoo-v1` not `butt2-v1`? And do you want `-v1`?

**Comment by arj03** [comment]
What the hell. This got stuck in a wierd state, maybe because the repo was moved?

**Pull request closed without merging**